docs/technical-design/launch-darkly-testing-approach.md

```diff
@@ -1,15 +1,149 @@
+These implementations are heavily relent on the data and generated Types in `flags.ts` located in `app-web/src/common-code/featureFlags`. This is done for several reasons:
```

Comment:
```suggestion
These implementations are heavily reliant on the data and generated Types in `flags.ts` located in `app-web/src/common-code/featureFlags`. This is done for several reasons:
```

Reply:
I really like the choice to move this to the top for conceptual grounding ⭐